contracts/external/interestsetters/DoubleExponentInterestSetter.sol

```diff
@@ -119,14 +119,19 @@ contract DoubleExponentialInterestSetter is
             });
         }
 
-        uint256 result = 0;
-        uint256 polynomial = BASE;
+        // process the first coefficient
+        uint256 coefficients = s.coefficients;
+        uint256 result = uint8(coefficients) * BASE;
+        coefficients >>= BYTE;
+
+        // initialize polynomial as the utilization
+        // no safeDiv since supplyWei must be non-zero at this point
+        uint256 polynomial = BASE.mul(borrowWei) / supplyWei;
 
         // for each non-zero coefficient...
-        uint256 coefficients = s.coefficients;
         while (true) {
             // gets the lowest-order byte
-            uint256 coefficient = coefficients % 256;
+            uint256 coefficient = uint256(uint8(coefficients));
```

Comment:
any reason you switched the way you're getting the lowest order byte?

Reply:
Just more readable imo, same gas cost